stable_diffusion_videos/app.py

```diff
@@ -60,7 +60,7 @@ def fn_videos(
         guidance_scale=guidance_scale,
         prompts=prompts,
         seeds=seeds,
-        num_steps=num_walk_steps,
+        num_interpolation_steps=num_interpolation_steps,
```

Comment:
Needs to be changed in function definition as well! After that should be good here.